package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "dmark",
-  "version": "1.0.7",
+  "version": "1.1.0-alpha1",
```

Add correct release tag.
```suggestion
  "version": "1.1.0",
```

---

added